pgrx-pg-sys/build.rs

```diff
@@ -742,6 +742,9 @@ fn add_blocklists(bind: bindgen::Builder) -> bindgen::Builder {
         .blocklist_type("Oid") // "Oid" is not just any u32
         .blocklist_function("varsize_any") // pgrx converts the VARSIZE_ANY macro, so we don't want to also have this function, which is in heaptuple.c
         .blocklist_function("(?:raw_)?(?:query|expression)_tree_walker")
+        .blocklist_function("query_or_expression_tree_walker")
+        .blocklist_function("planstate_tree_walker")
+        .blocklist_function("range_table_(?:entry_)?walker")
```

Comment:
So I wouldn't mind this being two or three lines but four is out:

```suggestion
        .blocklist_function("(?:raw_)?(?:expression|planstate|query|query_or_expression|range_table)(?:_entry)?_tree_walker")
```

Reply:
Disclaimer: I hate regular expressions as much as they hate me... So I'm always inclined to write a bunch of simple regexps rather than a single sophisticated one. 

But I get your point of keeping the blocklist small... 

However, if i'm correct your regexp would block a lot of false positives such as  `raw_planstate_entry_tree_walker`. There's little chance such function would ever be implemented in Postgres, but by principle shouldn't we match the strict minimum list of possibilities ?

Also it doesn't catch `range_table_entry_walker`

How about those 3 lines ?

```rust
        .blocklist_function("(?:raw_)?(?:query|expression)_tree_walker")
        .blocklist_function("(?:planstate|query_or_expression)_tree_walker")
        .blocklist_function("range_table_(?:entry_)?walker")
```

Reply:
Ah, I didn't notice `range_table_entry_walker` didn't have the terminal `tree`.

That's fine.

Reply:
```suggestion
        .blocklist_function("(?:raw_)?(?:query|expression)_tree_walker")
        .blocklist_function("(?:planstate|query_or_expression)_tree_walker")
        .blocklist_function("range_table_(?:entry_)?walker")
```